MoltenVK/MoltenVK/GPUObjects/MVKDevice.h

```diff
@@ -678,6 +686,7 @@ class MVKDevice : public MVKDispatchableVulkanAPIObject {
 	const VkPhysicalDeviceScalarBlockLayoutFeaturesEXT _enabledScalarLayoutFeatures;
 	const VkPhysicalDeviceTexelBufferAlignmentFeaturesEXT _enabledTexelBuffAlignFeatures;
 	const VkPhysicalDeviceVertexAttributeDivisorFeaturesEXT _enabledVtxAttrDivFeatures;
+	const VkPhysicalDevicePrivateDataFeaturesEXT _enabledPrivateDataFeatures;
```

Comment:
I like to keep these sorted like they are in MVKExtensions.def.

Reply:
Done.